{"name": "困人处置列表-真实", "status": "passed", "attachments": [{"name": "log", "source": "407a9c27-b1e0-465e-ac6f-b6e30c252365-attachment.txt", "type": "text/plain"}], "start": 1692326135522, "stop": 1692326135925, "uuid": "2be7ee10-b0f6-4876-b77c-f83663af90cc", "historyId": "2c9991b785716977eb4cbbaa1f323277", "testCaseId": "792deb5b60e48cc51a654ad1e9657ccb", "fullName": "opsPC.test_emergency#test_device_error_trapped_list_003", "labels": [{"name": "feature", "value": "运维管理平台"}, {"name": "story", "value": "应急处置"}, {"name": "parentSuite", "value": "opsPC"}, {"name": "suite", "value": "test_emergency"}, {"name": "host", "value": "cale"}, {"name": "thread", "value": "8932-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "opsPC.test_emergency"}]}